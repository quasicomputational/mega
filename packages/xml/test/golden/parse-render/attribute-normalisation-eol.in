<foo bar="hello  what? 
okay &#xD; bye" />