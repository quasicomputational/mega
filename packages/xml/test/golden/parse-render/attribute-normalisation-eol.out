<foo bar="hello   what?  okay &#xD; bye"></foo>